src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/RolesController.kt

```diff
@@ -289,7 +290,17 @@ data class CreateRole(
   )
   @field:NotEmpty(message = "Admin type cannot be empty")
   val adminType: Set<AdminType>,
-)
+) {
+  companion object {
+    const val ROLE_PREFIX = "ROLE_"
+    fun removeRolePrefixIfNecessary(role: String): String =
+      if (StringUtils.startsWith(role, ROLE_PREFIX)) StringUtils.substring(role, ROLE_PREFIX.length) else role
```

Comment:
don't think you need `StringUtils` to do any of this - should just be able to do it with kotlin string functions